lib/go-tc/deliveryservices.go

```diff
@@ -179,6 +189,13 @@ type DeliveryServiceNullableV30 struct {
 	ServiceCategory    *string `json:"serviceCategory" db:"service_category"`
 }
 
+type DeliveryServiceV31 struct {
+	DeliveryServiceV30
+	MaxRequestHeaderSize *int `json:"maxRequestHeaderSize" db:"max_request_header_size"`
```

Comment:
Would you object to changing this to `MaxRequestHeaderBytes`?
I think it makes it easier, so people don't have to look up the documentation to know the units, and don't assume the wrong unit and make a mistake.